### ISSUE TITLE
plugin v 1.2.7 cannot be activated due to fatal error

### ISSUE THREAD

**Issue by JackDougherty** [BCcampus/pressbooks-textbook#20]
Hi Brad,

I received the same fatal error on two test sites when attempting to update Pressbooks Textbook to the newest v 1.2.7, with PB v 2.4.5 (from WP repo) and WP 4.2.2.

Fatal error: Cannot redeclare class PressBooks\Import\Import in /var/www/pressbookstest/wp-content/plugins/pressbooks/includes/modules/import/class-pb-import.php on line 17

Did I screw up something obvious?
Jack

**Comment by JackDougherty** [comment]
Oddly, everything seems to be working on a 3rd test site, so maybe I screwed up something. Will explore further.

**Comment by bdolor** [comment]
please let me know how it goes, Jack.

**Comment by JackDougherty** [comment]
Brad, this is odd behavior that I cannot replicate across what i believe are identical test sites.

On one test site (http://pressbookstest2.trinfocafe.org) I successfully installed PB 2.4.5 and PBT 1.2.7 (both from WP repo) on WP 4.2.2.

But on two other test sites (including http://pressbookstest.trinfocafe.org, which runs on the same server as the one above), the PBT 1.2.7 causes a fatal error on activation. 

On both of the unsuccessful sites, I tried:
- deactivating all other plugins
- using SSH to manually delete the old plugin directories for PB and PBT

No dice. I'll look at this again later tonight or tomorrow.

**Comment by bdolor** [comment]
thanks Jack. I will also look into this and see if there is anything I can add to prevent this type of scenario from even being possible.

**Comment by JackDougherty** [comment]
Here's more info: on both of the test sites that were unsuccessful yesterday, I successfully installed PBT 1.2.6 (as a zip file from your GitHub release) with PB 2.4.5 on WP 4.2.2.

But when I tried to "update now" from PBT 1.2.6 to 1.2.7 via the WP repo (see screenshot), the same fatal error occurred. Something seems strange about the 1.2.7 release, at least on my end, for two of my three test sites.

![plugins_ _network_admin__pressbookstest_ _pressbooks](https://cloud.githubusercontent.com/assets/649719/7859288/78c42f5e-050d-11e5-9a99-88e759b188af.png)
